lte/protos/oai/nas_state.proto

```diff
@@ -419,5 +426,7 @@ message EmmContext {
   uint32 ksi = 64;
 
   UeNetworkCapability ue_network_capability = 65;
+  Timer t3422 = 66;
```

Comment:
No need to save the t3422 timer as it is irrelevant when the service restarts (the timer IDs do not persist after restart). 

ToDo: clean-up other NAS timers in state

Reply:
Timer contains both timer-id and timer-duration. Ya since timer-ids are not persistent so no need to save the timer-id. But we need to save timer-duration.